content/blog/2023/06/22/2023-06-22-mirrors-jenkins-new-IP.adoc

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "New Public IPv4 for Jenkins Mirrors"
+tags:
+- infrastructure
+- mirrors
+- jenkins
+authors:
+- dduportal
+opengraph:
+  image: /images/post-images/2023/01/12/jenkins-newsletter/infrastructure.png
+links:
+  discourse: https://community.jenkins.io/t/new-public-ipv4-for-jenkins-mirrors/8137
+---
+
+The Jenkins project provides a download mirror infrastructure allowing to download Jenkins packages and plugins from a download server close to your location.
+
+It provides a "HTTP redirector" service hosted behind the domains `get.jenkins.io`, `mirrors.jenkins.io` and `mirrors.jenkins-ci.org` which has a new public IP since the 12 June 2023.
+
+Please update your DNS servers and firewall rules to the new IP `20.119.232.75` if you have a restricted environment.
+
+The former redirector service and its previous IPv4 will be decommissioned the 27 of June 2023.
+
+More details in <https://github.com/jenkins-infra/helpdesk/issues/3351>.
```

Comment:
```suggestion
More details in link:https://github.com/jenkins-infra/helpdesk/issues/3351[the helpdesk issue].
```

Reply:
I'm not a fan of hidden links by a text: it does not provide any actionnable for the reader and if the generated HTML does not provide the proper ALT and stuff, then it will be hard for users requiring some kind of accssibility (while a full raw URL is always treated properly).
The `link` macro in Asciidoctor is not mandatory with full URL.

However the "chevrons" are an error from me, as asciidoctor does not support it.

Reply:
```suggestion
More details in https://github.com/jenkins-infra/helpdesk/issues/3351.
```